plugin/src/main/java/com/h3xstream/findsecbugs/taintanalysis/TaintFrameModelingVisitor.java

```diff
@@ -190,6 +190,8 @@ public void visitSIPUSH(SIPUSH obj) {
 
     @Override
     public void visitGETSTATIC(GETSTATIC obj) {
+        // Scala uses some classes to represent null instances of objects
+        // If we find one of them, we will handle it as a Java Null
         if (obj.getLoadClassType(getCPG()).getSignature().equals("Lscala/collection/immutable/Nil$;")) {
```

Comment:
What are the benefit of handling this correspondence? Are there any detector using this?
Nil could be added to the IMMUTABLE types .. https://github.com/find-sec-bugs/find-sec-bugs/blob/master/plugin/src/main/resources/taint-config/java-lang.txt